src/universalSortPoCGONEWRONG.java

```diff
@@ -0,0 +1,302 @@
+import java.util.ArrayList;
+import java.util.Collections;
+
+import components.map.Map;
+import components.map.Map1L;
+import components.queue.Queue;
+import components.queue.Queue1L;
+import components.sequence.Sequence;
+import components.sequence.Sequence1L;
+import components.set.Set;
+import components.set.Set1L;
+import components.simplewriter.SimpleWriter1L;
+import components.stack.Stack;
+import components.stack.Stack1L;
+
+/**
+ * Proof of concept for the universalSort class with a handful of methods, some
+ * example fields and a demonstration of how it would work
+ *
+ * @author Dev Morker
+ *
+ */
+public class universalSortPoCGONEWRONG {
```

Comment:
Okay, I think I generally get the idea. The fact that you have two files is a little confusing, but the premise makes sense. You want to be able to sort all the data structures, even if they are not sortable. I think that's a reasonable idea. I am not entirely sure of the purpose of the maps in this file. The other file is much simpler and stores booleans. I thought maybe this was secondary and the file below is kernel, but it's unclear.